packages/sundialsml/sundialsml.6.0.0p0/opam

```diff
@@ -0,0 +1,78 @@
+opam-version: "2.0"
+maintainer: "tim@tbrk.org"
+authors: [
+    "Timothy Bourke <tim@tbrk.org>"
+    "Jun Inoue <Jun.Lambda@gmail.com>"
+    "Marc Pouzet <Marc.Pouzet@ens.fr>"
+]
+homepage: "http://inria-parkas.github.io/sundialsml/"
+bug-reports: "https://github.com/inria-parkas/sundialsml/issues"
+dev-repo: "git+https://github.com/inria-parkas/sundialsml"
+doc: "http://inria-parkas.github.io/sundialsml/"
+tags: [
+    "numerical"
+    "simulation"
+    "mathematics"
+    "science"
+]
+license: "BSD-3-Clause"
+build: [
+    ["./configure" "--stubdir=%{stublibs}%/"
+		   "--libdir=%{lib}%/"
+		   "--docdir=%{doc}%/sundialsml/"
+		   "--os-distribution=%{os-distribution}%"]
+    [make "-j%{jobs}%"]
+    [make "doc"] {with-doc}
+]
+install: [
+  [make "install-findlib"]
+  [make "install-doc"] {with-doc}
+]
+remove: [
+    ["ocamlfind" "remove" "sundialsml"]
+    ["rm" "-rf" "%{doc}%/sundialsml"]
+]
+depends: [
+    "ocaml" {>= "4.03.0"}
```

Comment:
```suggestion
    "ocaml" {>= "4.08.0"}
```

the option module was introduced in 4.08. If you need to remain compatible with older releases you can use stdcompat to backport option to the old compilers